app/controllers/preprints/provider/preprint-detail.js

```diff
@@ -124,6 +124,7 @@ export default Controller.extend({
         },
         submitDecision(trigger, comment, filter) {
             this.toggleProperty('savingAction');
+            this.set('userHasEnteredReview', false);
```

Comment:
`ember-onbeforeunload` has a function that you can call in the router called `shouldCheckIsPageDirty`, where you can exclude certain routes from the `isPageDirty` check.  In normal preprints, we have the submit route/button exempt from the warning modal on click.  

Would it be a better practice to use `shouldCheckIsPageDirty` here too instead of changing `userHasEnteredReview` in the controller?